openstudiocore/src/shared_gui_components/EditView.cpp

```diff
@@ -121,6 +134,7 @@ EditRubyMeasureView::EditRubyMeasureView(bool applyMeasureNow)
 
   if(applyMeasureNow){
     nameLineEdit->setReadOnly(true);
+    nameNonEditableLineEdit->setVisible(false);
```

Comment:
@jmarrec just curious, why not show this as read only in apply measure now?

Reply:
Yeah I don't know either, let me change that real quick. (plus, I would actually need to hide the title too if hide the input box)